yarn.lock

```diff
@@ -1584,6 +1584,11 @@ async-each@^1.0.0:
   version "1.0.1"
   resolved "https://registry.yarnpkg.com/async-each/-/async-each-1.0.1.tgz#19d386a1d9edc6e7c1c85d388aedbcc56d33602d"
 
+async-limiter@^1.0.0:
```

Comment:
I've checked that all new deps are MIT cc @eclipse-theia/ip-help

Reply:
I did the recursive check of NPM production dependencies and everything looks good.

Reply:
@marcdumais-work thank you!